DeviceCode/pal/TouchPanel/touchpanel_driver.cpp

```diff
@@ -460,8 +460,11 @@ HRESULT TouchPanel_Driver::GetSetTouchInfo(UINT32 flags, INT32* param1, INT32* p
             // Touch Move events are queued up and sent at the given time frequency (StylusMoveFrequency)
             // We should not set the move frequency to be less than the sample frequency, otherwise there
             // would be no data available occassionally.
-            else if(ticks < min_ms_per_touchsample) return CLR_E_OUT_OF_RANGE;
-            else                                    ticks = TOUCH_PANEL_SAMPLE_MS_TO_TICKS(ms_per_touchmove_event);
+            else

```

Comment:
This appears to be a change in logic not a warning fix. Can you explain what the problem was and how this fixes it?